src/psyclone/tests/psyGen_test.py

```diff
@@ -4057,8 +4125,49 @@ class UnrecognizedType(object):
     assert fake_parent.symbol_table.lookup("array3").name == "array3"
     assert fake_parent.symbol_table.lookup("array3").datatype == 'real'
     assert fake_parent.symbol_table.lookup("array3").shape == [None]
-    assert fake_parent.symbol_table.lookup("array3").scope == "global_argument"
-    assert fake_parent.symbol_table.lookup("array3").is_input is True
+    assert fake_parent.symbol_table.lookup("array3").scope == "global"
+    assert fake_parent.symbol_table.lookup("array3").access is \
+        Symbol.Access.READ
+
+
+def test_fparser2astprocessor_use(f2008_parser):
+    ''' Check that SymbolTable entries are correctly created from
+    module use statements. '''
+    from fparser.common.readfortran import FortranStringReader
+    from fparser.two.Fortran2003 import Specification_Part
+    fake_parent = KernelSchedule("dummy_schedule")
+    processor = Fparser2ASTProcessor()
+    reader = FortranStringReader("use my_mod, only: some_var\n"
+                                 "use this_mod\n"
+                                 "use other_mod, only: var1, var2\n")
+    fparser2spec = Specification_Part(reader)
+    processor.process_declarations(fake_parent, fparser2spec.content, [])
+    for var in ["some_var", "var1", "var2"]:
+        assert fake_parent.symbol_table.lookup(var).name == var
+        assert fake_parent.symbol_table.lookup(var).scope == "global"
+    assert fake_parent.symbol_table.lookup("some_var").interface.module_name \
+            == "my_mod"
```

Comment:
pycodestyle says `continuation line over-indented for visual indent`

Reply:
Done.